### PR TITLE
get ride of service-specific notions

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

File: draft-ldbc-cats-framework.md [hunk 1]
```diff
@@ -402,7 +402,7 @@ This document does not specify any algorithm for path computation and selection
...
+Access to a service consists of one or more service-specific packets (e.g., Session Initiation Protocol (SIP) {{?RFC3261}}, HTTP {[?RFC9112}}, Real-Time Streaming Protocol (RTSP) {{?RFC7826}}) sent by the client via an Ingress CATS-Forwarder to which the client is connected to. The Ingress CATS-Forwarder classifies incoming packets received from clients by soliciting the CATS classifier (C-TC). When a matching classification entry is found for the packets, the Ingress CATS-Forwarder encapsulates and forwards them to the C-PS selected Egress CATS-Forwarder. Then, when these packets reach the Egress CATS-Forwarder, the outer header of the possible overlay encapsulation is removed and inner packets are sent to the relevant service contact instance.
```
Comment (VMatrix1900) [inline draft-ldbc-cats-framework.md:405]: "Access to a service" looks like a procedure to me. How about using service request, same as the title of the section?

Comment (boucadair) [reply draft-ldbc-cats-framework.md:405]: Issuing service request is just an example of accessing a service.

Comment (VMatrix1900) [reply draft-ldbc-cats-framework.md:405]: Is there other way to access the service without issuing the service request?

Comment (boucadair) [reply draft-ldbc-cats-framework.md:405]: I agree that's likely,  but I do still need to think about services that separate sig vs. data channels out there to generalize. 

For now, it is not harmful to have service access as a generic concept and then assume that access is likely to be triggered by a service request.

Comment (VMatrix1900) [reply draft-ldbc-cats-framework.md:405]: Right. Service with separate sig and data channel may be an outlier. Service access is fine. Do we need to change the section title to service access too?

Comment (boucadair) [reply draft-ldbc-cats-framework.md:405]: > Do we need to change the section title to service access too?

I think so. see [7fa896e](https://github.com/boucadair/CATS-framework/pull/47/commits/7fa896eb355dd49855f2b98144ef36f2186f8839)

**Review by VMatrix1900 — APPROVED** [review]

**Review by muzixing — APPROVED** [review]

**Pull request merged**